omero/sysadmins/unix/install-web/web-deployment.txt

```diff
@@ -35,7 +35,7 @@ Prerequisites
    `nginx <http://nginx.org/>`_ and `gunicorn <http://docs.gunicorn.org/>`_,
 
 
-Install the above OMERO.web dependences using the package management tools:
+Install the above OMERO.web dependencies using the package management tools:
```

Comment:
The ordering here is a bit confusing, if the virtualenv bit needs to come before the installation of the dependencies, it might be clearer to have:
"Install the above OMERO.web dependencies using the package management tools (virtualenv is an optional first step):"